modules/virt-about-storage-volumes-for-vm-disks.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/about-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-storage-volumes-for-vm-disks_{context}"]
+= About storage volumes for virtual machine disks
+
+If you use the storage API with known storage providers, volume and access modes are selected automatically. However, if you use a storage class that does not have a storage profile, you must select the volume and access mode.
+
+For best results, use `accessMode: ReadWriteMany` and `volumeMode: Block`. This is important for the following reasons:
+
+* The ReadWriteMany (RWX) access mode is required for live migration.
+
+* The `Block` volume mode performs significantly better in comparison to the `Filesystem` volume mode. This is because the `Filesystem` volume mode uses more storage layers, including a file system layer and a disk image file. These layers are not necessary for VM disk storage.
+
+For example, if you use {rh-storage-first}, Ceph RBD volumes are preferable to CephFS volumes.
+
+[IMPORTANT]
+====
+You cannot live migrate virtual machines that use:
+
+* A storage class with ReadWriteOnce (RWO) access mode
```

Comment:
Did you intend to write "storage class"? It might be safer to use "storage volume" so that the user does not get bogged down in storage class issues like setting default storage class.

I think this info might be more readable as a single line: "A virtual machine with a ReadWriteOnce storage volume or a passthrough device, such as vGPU, cannot be live migrated."

I wonder whether the second sentence is necessary. IIRC, the user cannot set the evictionStrategy to LiveMigrate in the UI if the VM that cannot be live migrated.